parsigs/parse_sig_api.py

```diff
@@ -86,27 +87,39 @@ def _parse_sig(sig: str, model: Language):
 
     return _create_structured_sigs(model_output)
 
+def _autocorrect(sig):
+    spell = SpellChecker()
+    spell.word_frequency.load_text_file(os.path.join('data', 'drug_names.txt'))
+    corrected_words = [spell.correction(word) if not any(char.isdigit() for char in word) else word for word in sig.split()]
```

Comment:
this is too much code in one line :) tru seperating the logic including function definitions to improve readability

Reply:
Fixed.